py/API_operations/imports/Import.py

```diff
@@ -341,6 +342,10 @@ def do_real(self) -> None:
 
         # Update loaded files in DB, removing duplicates
         self.prj.fileloaded = "\n".join(set(import_how.loaded_files))
+
+        # Check if we have a CNN model selected
+        feature_extractor_selected = self.prj.cnn_network_id != ""
```

Comment:
La colonne est null-able donc ici on peut se retrouver avec True si on n'a rien.
ecotaxa=# select count(1) from projects where cnn_network_id is null;
 count 
-------
  2057

Reply:
J'ai debuggue et vérifié la condition, par contre je ne suis pas sur qu'on soit dans cette etat en prod. Je ne sais pas s'il faut faire quelquechose de mieux.